frontend/src/components/AddIngredients.js

```diff
@@ -47,22 +49,36 @@ export default function AddIngredients({
         getOptionLabel={(ingredient) => ingredient.name}
         renderInput={(params) => <TextField {...params} label="Zutaten:" />}
       />
-      <button disabled={!ingredient}>Hinzufügen</button>
+      <BottomNavigationAction
+          className="addbutton"
+          color={"white"}
+          disabled={!description}
+          onClick={handleAddClick}
+          icon={<AddIcon style={{ color: "white" }} />}
+      />
     </Wrapper>
   );
 }
 
 const Wrapper = styled.form`
   display: flex;
+  flex-direction: column;
   justify-content: center;
+  align-items: center;
   padding: 24px;
+  
   input {
     margin-right: 12px;
     flex-grow: 1;
   }
   button {
     background-color: #008080;
     color: white;
-    border-radius: 12px;
   }
-`;
+  .addbutton {
+   margin-top: 24px;
+   border-radius: 12px;
+   min-width: 8px;
+   padding: 8px !important;
```

Comment:
`!important`is not good practice. You should consider to look for another solution :)

Reply:
Didn't found another solution ^^